src/osd-field.c

```diff
@@ -142,7 +144,9 @@ field_set_win_state_all(struct buf *buf, struct view *view, const char *format)
 	/* custom type conversion-specifier: S */
 	buf_add(buf, view->minimized ? "m" : " ");
 	buf_add(buf, view->maximized ? "M" : " ");
-	buf_add(buf, view->fullscreen ? "F" : " ");
+	// XXX WIP -- "s" should be handled separately - consumes space in
+	// XXX custom field -- harder is to choose order...
+	buf_add(buf, view->fullscreen ? "F" : view->shaded ? "s" : " ");
```

Comment:
I think `s` should just be a usual field here. The difference between the two `set_win_state()` is that one only shows a single character indicating state while the other shows all states. Suddenly having fullscreen and shaded share a single character while all other states use a dedicated one sounds incoherent to me.